docs/source/markdown/podman-stop.1.md

```diff
@@ -25,6 +25,14 @@ Stop all running containers.  This does not include paused containers.
 
 Read container ID from the specified file and remove the container.  Can be specified multiple times.
 
+**--allow-missing**
+
+Ignore errors when specified containers are not in the container store. The
+motivation behind this flag is to write more robust systemd services using
+Podman. A user might have manually decided to remove a container which would
+lead to a failure during the ExecStop directive of a systemd service referencing
+that container.
+
```

Comment:
My tilt would be to remover the history lesson, i.e. remove the second sentence here that begins with "The motivation ...".  The third sentence is really good though.  I don't think it's important to document our reasons for providing a flag, rather we should document why the flag might be important for the user to use.  Especially so on a flag whose name when first read causes the user to question why/how it would be used.

Actually suggested tweak on the third sentence.  s/manually decided to/decided to manually/

Reply:
done